international-labour-organization.csl

```diff
@@ -137,16 +144,20 @@
         <date-part name="year"/>
       </date>
       <choose>
-        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <if type="book" match="any">
           <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
+            <text macro="title" font-style="italic"/>
```

Comment:
no need for the `font-style="italic"` here -- you're already ding that in the macro